values.yaml

```diff
@@ -0,0 +1,6 @@
+appName: loki-alert-rule-group-controller
+appNamespace: nil
+replicaCount: 1
+image:
+  pullPolicy: nil
```

Comment:
@fernandoataoldotcom i would just default this to always pull

Reply:
I set Always here, but can definitely move it to the upstream chart: https://github.com/GlueOps/platform-helm-chart-platform/blob/c5adc122f2bec3d2d183f56573b36eaefdad4412/templates/application-loki-alert-group-controller.yaml#L22

Reply:
I would just throw it in the upstream chart so it's one less thing to think about in the application spec